2022 - Legiones Astartes.cat

```diff
@@ -13898,7 +13898,7 @@ This is a Damage Mitigation roll – any model may make only a single Damage Mit
           </profiles>
           <infoLinks>
             <infoLink id="4383-5ee3-a507-d62e" name="Relentless" hidden="false" targetId="7adf-ac9a-5035-522d" type="rule"/>
-            <infoLink id="21e0-a06e-2f9e-0b2d" name="Legiones Consularis" hidden="false" targetId="dcc5-b4e0-35c5-0c17" type="rule"/>
+            <infoLink id="21e0-a06e-2f9e-0b2d" name="Legiones Consularis" hidden="false" targetId="5a1d-d2f6-692b-d834" type="rule"/>
```

Comment:
Do we want to just remove this, since the rule also exists in the linked SSEG?

Reply:
I see what you mean, and I've removed it